app/javascript/Component/Greetings.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchGreeting } from "../Redux/greetingsSlice";
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greetings.greeting);
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  return <h2>{greeting}</h2>;
+};
```

Comment:
- [ ] Please kindly ensure that you create a table for storing your messages and create 5 different greetings in your `db/seed.rb` file. 🙏 

<img width="629" alt="Untitled" src="https://github.com/ismayito/hello-rails-react/assets/103386183/74cbd87e-ce16-4b50-9618-357e497c132f">